### PR TITLE
Final Request for Alerts

### PR DESCRIPTION
**Pull request by depth-dev**
Added MiniAlerts.

Also added "Stonk Dragon" to trivia.json as per request from bug reports.

### CODE REVIEW

File: features/miniAlert.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+class miniAlert {
+    constructor() {
+        this.name = "MiniBoss Alerts"
+        this.triggers = {
+            "renderWorld": [{
+                "func": this.renderWorld,
+                "criteria": null
+            }],
```
Comment (Antonio32A) [inline features/miniAlert.js]: remove the comma

File: features/miniAlert.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        return;
+    }
+    Scoreboard.getLines(false).forEach(line => {
+        if (line.getName().includes('§cThe Cata') && line.getName().includes('mbs §8('));
+        return true;
+    })
+    return false;
+}
+
+class miniAlert {
```
Comment (Antonio32A) [inline features/miniAlert.js]: should be called miniBossAlerts

File: features/miniAlert.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+const inDungeons = () => {
+    if (!(Scoreboard.getTitle().trim() != "" && Scoreboard.getLines().length > 0)) {
+        return;
+    }
+    Scoreboard.getLines(false).forEach(line => {
+        if (line.getName().includes('§cThe Cata') && line.getName().includes('mbs §8('));
```
Comment (Antonio32A) [inline features/miniAlert.js]: double quotes

**Review by Antonio32A — CHANGES_REQUESTED** [review]

**Comment by ElOxo** [comment]
Dont forget to Fix Deap Cavern to Deap Caverns in trivia.json

**Comment by Dolphin0xyz** [comment]
You might need to add frozen adventurer to miniAlert.js

**Pull request closed without merging**